src/components/Trade/types.ts

```diff
@@ -71,25 +69,13 @@ export type SupportedSwappingChain =
   | KnownChainIds.OsmosisMainnet
   | KnownChainIds.CosmosMainnet
 
-type GetFirstReceiveAddressArgs = {
-  accountSpecifiersList: ReturnType<typeof selectAccountSpecifiers>
-  buyAsset: Asset
-  chainAdapter: ChainAdapter<ChainId>
-  wallet: HDWallet
+export type GetReceiveAddressArgs = {
+  asset: Asset
+  wallet: HDWallet | null
   bip44Params: BIP44Params
-  accountType: UtxoAccountType | undefined
+  accountType?: UtxoAccountType
```

Comment:
Shouldn't this be 
```suggestion
  accountType?: UtxoAccountType | undefined
```

As callers explicitly pass undefined, see

https://github.com/shapeshift/web/blob/bb5f3451eff7fa0b31f140aa12d06623544d452c/src/components/Trade/hooks/useSwapper/useSwapperV2.tsx#L96-L101